src/main/java/com/example/lecture09task/controller/UserController.java

```diff
@@ -48,4 +53,34 @@ public ResponseEntity<Map<String, String>> update(
         userService.updateUser(form.convertToUser(id));
         return ResponseEntity.ok(Map.of("message", "user successfully updated"));
     }
+
+    @DeleteMapping("/users/{id}")
+    public ResponseEntity<Map<String, String>> delete(@PathVariable("id") int id) {
+        userService.deleteUser(id);
+        return ResponseEntity.ok(Map.of("message", "user successfully deleted"));
+    }
+
+    @ExceptionHandler(value = ResourceNotFoundException.class)
+    public ResponseEntity<Map<String, String>> handleNoResourceFound(
+            ResourceNotFoundException e, HttpServletRequest request) {
+        Map<String, String> body = Map.of(
+                "timestamp", ZonedDateTime.now().toString(),
+                "Status", String.valueOf(HttpStatus.NOT_FOUND.value()),
```

Comment:
全体的に小文字のみなので小文字で統一したいですね。
```suggestion
                "status", String.valueOf(HttpStatus.NOT_FOUND.value()),
```

Reply:
確かにここだけ大文字になっていました。。。
下記コメントと合わせて修正しました！
78fd896e8ef1cf3a66868fc4729e2d1b48b8d99e